lib/rules/link-rel-noopener.js

```diff
@@ -77,17 +81,47 @@ module.exports = class LinkRelNoopener extends Rule {
           return;
         }
 
-        this.log({
-          message: this.config.message,
-          line: node.loc && node.loc.start.line,
-          column: node.loc && node.loc.start.column,
-          source: this.sourceForNode(node),
-        });
+        if (this.mode === 'fix') {
+          fix(node, this.config);
+        } else {
+          this.log({
+            isFixable: true,
```

Comment:
looks like we able to fix it if rel attribute value is a textNode, in other cases, it should be false

Reply:
@lifeart thanks! great catch. updated